src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+const big100 = Big(100);
+// const bigZero = Big(0);
+// const bigOne = Big(1);
+
+const operate = (operation, numOne, numTwo) => {
+  numOne = Big(numOne);
+  numTwo = Big(numTwo);
+  const operateWith = {
+    '+': (numOne, numTwo) => numOne.plus(numTwo).toString(),
+    '-': (numOne, numTwo) => numOne.minus(numTwo).toString(),
+    '*': (numOne, numTwo) => numOne.times(numTwo).toString(),
+    '/': (numOne, numTwo) => numOne.div(numTwo).toString(),
+    '%': (numOne) => numOne.div(big100),
```

Comment:
Please use the mod function in BigInt for the % operator, I see what you are trying to do but for now mod is the required functionality ![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)